exporter/opentelemetry-exporter-otlp/setup.cfg

```diff
@@ -36,24 +36,8 @@ classifiers =
 
 [options]
 python_requires = >=3.5
-package_dir=
-    =src
 packages=find_namespace:
 install_requires =
-    grpcio >= 1.0.0, < 2.0.0
-    googleapis-common-protos ~= 1.52.0
     opentelemetry-api == 1.0.0.dev0
```

Comment:
Do api and sdk still need to be dependencies?

Reply:
They do not, removed